python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/test_stubbed_ecs.py

```diff
@@ -272,6 +275,14 @@ def test_run_task(ecs, subnet):
     )
     assert response["tasks"][0]["overrides"]["containerOverrides"][0]["command"] == ["ls"]
 
+    # fails if old ARN schema is used and `propagateTags` parameter is set
+    if task_long_arn_format == "disabled":
+        with pytest.raises(ClientError):
+            ecs.run_task(taskDefinition="container", propagateTags="TASK_DEFINITION")
+    else:
+        with pytest.raises(ClientError):
+            ecs.run_task(taskDefinition="container", propagateTags="WRONG_PARAMETER")
```

Comment:
This does a good job of testing the various failure scenarios, but I don't think it adequately tests the happy path. I'd suggest something akin to:

```py
    # add some tags to a task definition - you might need to stub out some more
    # of `register_task_definition()` to make this happen

    # tags aren't propagated if the old ARN format is being used
    if task_long_arn_format == "disabled":
        with pytest.raises(ClientError):
            ecs.run_task(taskDefinition="container", propagateTags="TASK_DEFINITION")
            
    # tags aren't propagated if we pass an invalid value
    with pytest.raises(ClientError):
        ecs.run_task(taskDefinition="container", propagateTags="SERVICE")
    
    response = ecs.run_task(taskDefinition="container")
    assert not response["task"][0].get("tags")
    
    response = ecs.run_task(taskDefinition="container", propagateTags="TASK_DEFINITION")
    tags = response["tasks"][0].get("tags")
    # and then assert the tags match what you tagged the task definition with
```